.github/workflows/promote.yaml

```diff
@@ -11,7 +11,9 @@ on:
         required: true
         default: 'maintenance-3.0.x'
       dryRun:
-        description: 'Runs the workflow without pushing code or publishing artifacts. If empty, Dry Run is disabled.'
+        description: 'Run the workflow without pushing code or publishing artifacts'
+        type: boolean
```

Comment:
I think we need to update the check to see if the value is 'true' or 'false', right? I think we saw this before and there also https://github.com/actions/runner/issues/1483.

Reply:
Well, during all the tests I did, it seems that because the default value is `false` this doesn't constitute a problem for us, but we can check the value 👍